src/utils/editStepAttribute.ts

```diff
@@ -20,7 +20,7 @@ export const editStepAttribute = (
   const filteredAttribute = attribute.filter(
     (element: any) =>
       element !== 'OR' &&
-      element !== 'and' &&
+      element !== 'AND' &&
       element !== 'NOT' &&
       element !== 'or',
```

Comment:
is the lowercase correct?

Reply:
Yup, there was some rare case that forced us to add this condition.